<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-RJ73CUL4" failures="1" tests="8" name="Test123" time="0.093" errors="0" timestamp="2022-10-25T13:35:30 IST">
  <testcase classname="testData.TestngClass1" name="testA" time="0.006"/>
  <testcase classname="testData.TestngClass1" name="testB" time="0.002"/>
  <testcase classname="testData.TestNg$TestngClass" name="test1" time="0.001"/>
  <testcase classname="testData.TestNg$TestngClass" name="test2" time="0.003"/>
  <testcase classname="testData.TestNg$TestngClass" name="test3" time="0.008">
    <failure type="java.lang.AssertionError" message="Url of page not found expected [https://www.messe.com/help] but found [https://www.messenger.com/help]">
      <![CDATA[java.lang.AssertionError: Url of page not found expected [https://www.messe.com/help] but found [https://www.messenger.com/help]
at testData.TestNg$TestngClass.test3(TestNg.java:49)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase classname="testData.TestNg$TestngClass" name="test4" time="0.002"/>
  <testcase classname="testData.TestngClass2" name="test3" time="0.001"/>
  <testcase classname="testData.TestngClass2" name="test4" time="0.002"/>
</testsuite> <!-- Test123 -->
